deepspeed/runtime/zero/stage3.py

```diff
@@ -852,7 +852,7 @@ def __init__(self,
                 count = count + 1
 
         #Largest partitioned param
-        largest_partitioned_param_numel = max(self.fp16_partitioned_groups_flat_numel)
+        largest_partitioned_param_numel = max([max([tensor.numel() for tensor in fp16_partitioned_group]) for fp16_partitioned_group in self.fp16_partitioned_groups])

```

Comment:
Does self.fp16_partitioned_groups_flat_numel need to be updated?

Reply:
Do references like [this ](https://github.com/microsoft/DeepSpeed/blob/5f16b17297525b321208b493e89c665e68a04163/deepspeed/runtime/zero/stage3.py#L931) need updating?